protocol/x/prices/keeper/util_test.go

```diff
@@ -58,6 +58,12 @@ func TestGetProposalPrice(t *testing.T) {
 			marketPrice:   uint64(1_000_000),
 			expectedPrice: uint64(900_000),
 		},
+		"smoothedPrice: smoothedPrice < marketPrice < indexPrice": {
```

Comment:
Test cases to check that in condition where market price is equidistant from smoothed price and index price, smoothed price is chosen.